web/pdf_find_controller.js

```diff
@@ -322,10 +322,9 @@ var PDFFindController = (function PDFFindControllerClosure() {
      * @param {number} index - match index.
      * @param {Array} elements - text layer div elements array.
      * @param {number} beginIdx - start index of the div array for the match.
-     * @param {number} endIdx - end index of the div array for the match.
```

Comment:
We basically want to show entire match. If scrollIntoView would accept multiple elements then showing something elements[beginIdx] through elements[endIdx] will be a better choice. Not sure what will be a right call here. I'll let you decide :)